src/features/filter/filter.selectors.ts

```diff
@@ -22,7 +22,11 @@ export const { selectAll: selectFilters, selectById: selectFilterById } = filter
 export const selectClassFilterByPO = createSelector(
   (state: RootState, id?: PageObjectId) => {
     const pageObject = !isNil(id) ? selectPageObjById(state, id) : selectCurrentPageObject(state);
-    const filter = selectFilters(state).find(({ pageObjectId }: Filter) => pageObjectId === pageObject?.id);
+    const savedFilters = localStorage.getItem("filters");
```

Comment:
selectors is a thing only for getting data from state. Using them for something else potentially lead to bugs/ Selector is a memoized function under the hood, so we can face with case, when it doesn't retrieve an actual data from local storage.

Reply:
fixed